pkgs/development/python-modules/ocrmypdf/default.nix

```diff
@@ -31,7 +31,7 @@
 
 buildPythonPackage rec {
   pname = "ocrmypdf";
-  version = "15.4.0";
+  version = "15.4.3";
```

Comment:
Enable the PEP517 build here. No format means `python setup.py install`, which is deprecated.